TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Drivetrain.java

```diff
@@ -1,26 +1,45 @@
 package org.firstinspires.ftc.teamcode.subsystems;
 
 import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.IMU;
 
 import org.firstinspires.ftc.teamcode.hardware.RobotHardware;
 
 public class Drivetrain{
-    private final DcMotor motorFrontRight;
-    private final DcMotor motorFrontLeft;
-    private final DcMotor motorBackRight;
-    private final DcMotor motorBackLeft;
-
-    public Drivetrain(){
-        motorFrontRight = RobotHardware.motorFrontRight;
-        motorFrontLeft= RobotHardware.motorfrontLeft;
-        motorBackLeft= RobotHardware.motorbackLeft;
-        motorBackRight = RobotHardware.motorbackRight;
+
+    private static Drivetrain instance;
+    private static DcMotor MOTOR_RIGHT;
+    private static DcMotor MOTOR_LEFT;
```

Comment:
The pattern of THIS_TYPE_OF_VARIABLE is just for constants (keyword "final"). Since MOTOR_LEFT is being changed its values, you should just put "motorLeft"